docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Managing tracking scenarios in the console"
+sidebar_label: "🆕 Managing tracking scenarios in the console"
+sidebar_position: 95
+---
```

Comment:
@stanch I removed the date and set the 'new' in the sidebar_label. Let me know if I got it correct

Reply:
done